### ISSUE TITLE
Update the language bar

### ISSUE THREAD

**Issue by bear8642** [Dyalog/ride#1265]
### Describe the issue you are having
Update language bar to include behind

Requires mantis 21700 so tools/lb-gen can correctly update lb.js

**Comment by abrudz** [comment]
@bear8642 You self-assigned this. How is it going?

**Comment by bear8642** [comment]
Yes, self-assigned as thought it would be simple to do once [Mantis 21700](https://mantis.dyalog.com/view.php?id=21700) is completed.

The docs mantis issue is yet to be completed.
